osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/config/PersistenceConfigCore.java

```diff
@@ -21,28 +21,33 @@
 import org.springframework.orm.jpa.JpaTransactionManager;
 import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
 
-@EnableJpaRepositories(basePackageClasses = {DeviceRepository.class})
+@EnableJpaRepositories(
+    entityManagerFactoryRef = "coreEntityManagerFactory",
+    basePackageClasses = {DeviceRepository.class})
 @Configuration
 @PropertySource("classpath:osgp-adapter-ws-core.properties")
 @PropertySource(value = "file:${osgp/Global/config}", ignoreResourceNotFound = true)
 @PropertySource(value = "file:${osgp/AdapterWsCore/config}", ignoreResourceNotFound = true)
-public class PersistenceConfig extends AbstractPersistenceConfig {
+public class PersistenceConfigCore extends AbstractPersistenceConfig {
 
-  @Value("${db.readonly.username}")
+  @Value("${db.username.core}")
   private String username;
 
-  @Value("${db.readonly.password}")
+  @Value("${db.password.core}")
   private String password;
 
-  @Value("${db.host}")
+  @Value("${db.host.core}")
```

Comment:
What I meant here was that it would probably be better to rename the properties for the `WritablePersistenceConfig` class to for example: `db.host.core` as well, since it is used by the ws core adapter that now has a different property for the core db.